azure-devops/azext_devops/dev/boards/work_item.py

```diff
@@ -220,18 +220,22 @@ def show_work_item(id, open=False, organization=None, detect=None):  # pylint: d
 
 
 # pylint: disable=too-many-statements
-def query_work_items(wiql=None, id=None, path=None, organization=None, project=None, detect=None):  # pylint: disable=redefined-builtin
+def query_work_items(wiql=None, wiql_file_path=None, id=None, path=None, organization=None, project=None, detect=None):  # pylint: disable=redefined-builtin
     """Query for a list of work items.
     :param wiql: The query in Work Item Query Language format.  Ignored if --id or --path is specified.
     :type wiql: str
+    :param wiql_file_path: The query in Work Item Query Language format in a file.
+                           Ignored if --wiql, --id or --path is specified.
+                           Please use / backslash when typing in directory path.
+    :type wiql_file_path: str
     :param id: The UUID of an existing query.  Required unless --path or --wiql are specified.
     :type id: str
     :param path: The path of an existing query.  Ignored if --id is specified.
```

Comment:
1. What would be the priority /implementation order if both wiql and file path are specified - even though it looks to be an erroneous scenario, curious on how it is tackled?
2. Help text recommendation - "The local file path to a file containing the wiql"
3. Are there any nuances the user has to take care of when specifying path? I remember in one earlier path we were requesting for '/'. Is that the case? Further, is it just any "file" or are we expecting any particular file format - json/text?

Reply:
1. --wiql will be used and --wiql-file-path will be ignored. This is why this line is present `Ignored if --wiql, --id or --path is specified.`
2. Sure
3. yes. Please use / backslash when typing in directory path.  It is a file containing WIQL in plane text